myreadinglist/urls.py

```diff
@@ -14,7 +14,7 @@
     url(r'^books/', include('books.urls', namespace='books')),
     url(r'^users/(?P<username>.*)$', book_views.user_page, name='user_page'),
     url(r'^widget/(?P<username>.*)$', book_views.user_page_widget, name='user_page_widget'),
-    url(r'^accounts/', include('registration.backends.hmac.urls')),
+    url(r'^accounts/', include('registration.backends.activation.urls')),
```

Comment:
After add that line I get:

`ModuleNotFoundError: No module named 'registration'`

We need the latest: `path('accounts/', include('django_registration.backends.activation.urls')),`
https://django-registration.readthedocs.io/en/3.1.1/quickstart.html